### PR TITLE
[improve](mow) merge and remove old version of delete bitmap when cumulative compaction is done on local mode

### PR DESCRIPTION
**Pull request by hust-hhb**
pr https://github.com/apache/doris/pull/40204 remove delete bitmap on cloud mode, this pr is used for local mode

### CODE REVIEW

**Comment by hust-hhb** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

File: be/src/olap/compaction.cpp [hunk 1]
```diff
@@ -1045,6 +1072,12 @@ Status CompactionMixin::modify_rowsets() {
         tablet()->delete_expired_stale_rowset();
     }
 
+    if (compaction_type() == ReaderType::READER_CUMULATIVE_COMPACTION &&
```
Comment (zhannngchen) [inline be/src/olap/compaction.cpp:1075]: use config `config::enable_delete_bitmap_merge_on_compaction`

Comment (hust-hhb) [reply be/src/olap/compaction.cpp]: @zhannngchen done

**Comment by hust-hhb** [comment]
run buildall

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Comment by hust-hhb** [comment]
run buildall

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by hust-hhb** [comment]
run buildall

**Comment by hust-hhb** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by zhannngchen — APPROVED** [review]

**Pull request merged**